### DISCUSSION TITLE
[Q] Why pitch could be from 0 to 360 instead of -90 to +90?

### DISCUSSION THREAD

**Discussion by BlackYuzia** [saul/demofile#391] [Questions]
Hi there.

In ~~cago~~ CS:GO, `pitch` has range from `-90` to `+90`.
In demofile I get `pitch` from `0` to `360`.

That's fine? How I could transform it to default `pitch` range?

**Comment by saul** [comment]
Do you see pitch within the range [0, 360]? I would have only expected it to be in the range of [90, 270]. Then to convert to [-90, 90] range you subtract 180.

**Reply by BlackYuzia** [reply]
Yep, I check min & max value of pitch & yaw. Then get:
pitch min. = 0
pitch max. = 359.xxx (~360)

The same for yaw.
